packages/eth-sdk/src/abi-management/etherscan/urls.ts

```diff
@@ -1,82 +1,33 @@
 
-// note: copied from https://github.com/nomiclabs/hardhat/blob/master/packages/hardhat-etherscan/src/network/prober.ts
-export interface EtherscanURLs {
-  apiURL: string
-  browserURL: string
-}
-
-type NetworkId2Etherscan = {
-  [networkID in NetworkID]: EtherscanURLs
-}
+type NetworkId2Etherscan = { [networkID in NetworkID]: URLString }
 
+/**
+ * This object is adapted from hardhat-etherscan source.
+ * Refer to the following file to add new predefined networks:
+ *
+ * @see https://github.com/nomiclabs/hardhat/blob/master/packages/hardhat-etherscan/src/network/prober.ts
+ */
 export const networkIDtoEndpoints: NetworkId2Etherscan = {
-  [NetworkID.MAINNET]: {
-    apiURL: 'https://api.etherscan.io/api',
-    browserURL: 'https://etherscan.io',
-  },
-  [NetworkID.ROPSTEN]: {
-    apiURL: 'https://api-ropsten.etherscan.io/api',
-    browserURL: 'https://ropsten.etherscan.io',
-  },
-  [NetworkID.RINKEBY]: {
-    apiURL: 'https://api-rinkeby.etherscan.io/api',
-    browserURL: 'https://rinkeby.etherscan.io',
-  },
-  [NetworkID.GOERLI]: {
-    apiURL: 'https://api-goerli.etherscan.io/api',
-    browserURL: 'https://goerli.etherscan.io',
-  },
-  [NetworkID.KOVAN]: {
-    apiURL: 'https://api-kovan.etherscan.io/api',
-    browserURL: 'https://kovan.etherscan.io',
-  },
-  [NetworkID.BSC]: {
-    apiURL: 'https://api.bscscan.com/api',
-    browserURL: 'https://bscscan.com',
-  },
-  [NetworkID.BSC_TESTNET]: {
-    apiURL: 'https://api-testnet.bscscan.com/api',
-    browserURL: 'https://testnet.bscscan.com',
-  },
-  [NetworkID.HECO]: {
-    apiURL: 'https://api.hecoinfo.com/api',
-    browserURL: 'https://hecoinfo.com',
-  },
-  [NetworkID.HECO_TESTNET]: {
-    apiURL: 'https://api-testnet.hecoinfo.com/api',
-    browserURL: 'https://testnet.hecoinfo.com',
-  },
-  [NetworkID.OPERA]: {
-    apiURL: 'https://api.ftmscan.com/api',
-    browserURL: 'https://ftmscan.com',
-  },
-  [NetworkID.FTM_TESTNET]: {
-    apiURL: 'https://api-testnet.ftmscan.com/api',
-    browserURL: 'https://testnet.ftmscan.com',
-  },
-  [NetworkID.OPTIMISTIC_ETHEREUM]: {
-    apiURL: 'https://api-optimistic.etherscan.io/api',
-    browserURL: 'https://optimistic.etherscan.io/',
-  },
-  [NetworkID.OPTIMISTIC_KOVAN]: {
-    apiURL: 'https://api-kovan-optimistic.etherscan.io/api',
-    browserURL: 'https://kovan-optimistic.etherscan.io/',
-  },
-  [NetworkID.POLYGON]: {
-    apiURL: 'https://api.polygonscan.com/api',
-    browserURL: 'https://polygonscan.com',
-  },
-  [NetworkID.POLYGON_MUMBAI]: {
-    apiURL: 'https://api-testnet.polygonscan.com/api',
-    browserURL: 'https://mumbai.polygonscan.com/',
-  },
-  [NetworkID.ARBITRUM_ONE]: {
-    apiURL: 'https://api.arbiscan.io/api',
-    browserURL: 'https://arbiscan.io/',
-  },
-  [NetworkID.ARBITRUM_TESTNET]: {
-    apiURL: 'https://api-testnet.arbiscan.io/api',
-    browserURL: 'https://testnet.arbiscan.io/',
-  },
+  [NetworkID.MAINNET]: 'https://api.etherscan.io/api',
```

Comment:
My initial plan was to retain the original format of the config as it was in `hardhat-etherscan` to make updates easy.

Reply:
Would you like me to change this?

IMHO Copying two or three links manually every now and then shouldn't be that much work, and keeping that browserUrl leads to the question "where do we use it?"

Reply:
I don't have a strong opinion on this, just wanted to give some color to the original decision. I guess it also depends on the frequency of updates to `hardhat-etherscan`. Perhaps, in the near future, it will be extracted to a separate package so maybe we don't need to overthink it. To sum up: I am fine with your solution.